src/main/scala/uk/gov/nationalarchives/tdr/api/service/ConsignmentService.scala

```diff
@@ -29,13 +33,23 @@ class ConsignmentService(
   }
 
   def addConsignment(addConsignmentInput: AddConsignmentInput, userId: UUID): Future[Consignment] = {
-    val consignmentRow = ConsignmentRow(uuidSource.uuid, addConsignmentInput.seriesid, userId, Timestamp.from(timeSource.now))
-    consignmentRepository.addConsignment(consignmentRow).map(row => Consignment(row.consignmentid, row.userid, row.seriesid))
+    val now = timeSource.now
+    val yearNow = LocalDate.from(now.atOffset(ZoneOffset.UTC)).getYear
+     consignmentRepository.getNextConsignmentSequence.flatMap(sequence => {
```

Comment:
Something odd has happened to the indentation here.

Reply:
Oh yeah, I'll fix it! :+1: